### ISSUE TITLE
Text to Speech accessibility feature doesn't speak playlist names

### ISSUE THREAD

**Issue by rsn8887** [libretro/RetroArch#11854]
## Description

The Text-To-Speech accessibility feature works great on Windows, however, playlist names are not spoken out loud for some reason.

### Expected behavior

Playlist names, such as Mame-2003 Plus, should be spoken out when the cursor is over them, like anything else in the menu, when --accessibility is enabled via the command line.

### Actual behavior

Playlist names are not spoken out loud.

### Steps to reproduce the bug
0. start Retroarch with --accessibility command line option to enable text to speech
1. Import Mame roms
2. Select Mame 2003 Plus Playlist
3. Observe that the playlist name is not spoken out loud, while all other menu entries, including game names, are spoken out loud by the --accessibility feature when it is turned on

### Bisect Results

not done

### Version/Commit
latest nightly 01/08/2021

- RetroArch: latest nightly 01/08/2021

### Environment information

- OS: Windows 10
